xml/System.Runtime.Loader/AssemblyLoadContext.xml

```diff
@@ -259,9 +275,16 @@ This constructor is protected. It can only be called from the <xref:System.Runti
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Sets <see cref="P:System.Runtime.Loader.AssemblyLoadContext.CurrentContextualReflectionContext" /> to <see langword="this" /> <see cref="T:System.Runtime.Loader.AssemblyLoadContext" />.</summary>
+        <returns>A <see cref="T:System.Runtime.Loader.AssemblyLoadContext.ContextualReflectionScope" /> to restore the previous value of <see cref="P:System.Runtime.Loader.AssemblyLoadContext.CurrentContextualReflectionContext" />. Intended to be used as an <see cref="T:System.IDisposable" /> in a <see langword="using" /> block.</returns>
```

Comment:
```suggestion
        <returns>An object to restore the previous value of <see cref="P:System.Runtime.Loader.AssemblyLoadContext.CurrentContextualReflectionContext" />. It is intended to be used as an <see cref="T:System.IDisposable" /> in a <see langword="using" /> block.</returns>
```
I changed the return type name to "object" since the type name appears immediately above it.